.github/workflows/release.yml

```
@@ -133,51 +130,19 @@ jobs:
-        # v4
-        with:
-          name: ${{ env.REPO_NAME }}-${{ matrix.artifact }}
-          path: dist
+    uses: ./.github/workflows/build-and-package.yml
+    with:
+      platform: linux
+      runner: ubuntu-latest
+      target: ${{ matrix.target }}
+      bin-name: ${{ needs.metadata.outputs.bin_name }}
+      version: ${{ needs.metadata.outputs.version }}
+      artifact-name: ${{ env.REPO_NAME }}-${{ matrix.artifact }}
```

actionlint (1.7.8)

context "env" is not allowed here. available contexts are "github", "inputs", "matrix", "needs", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)